index.js

```diff
@@ -1,25 +1,42 @@
 'use strict';
 
 module.exports = function buildAirbnbPreset(context, options) {
-  var preset; // eslint-disable-line no-var
-  if (options && options.modules === false) {
-    preset = require('babel-preset-es2015').buildPreset(null, { modules: false });
-  } else {
-    preset = require('babel-preset-es2015-without-strict');
-  }
   return {
     presets: [
-      preset,
+      require('babel-preset-env').default(null, {
+        debug: true,
+        exclude: [
+          'transform-async-to-generator',
+          'transform-es2015-template-literals',
+          'transform-regenerator',
+        ],
+        modules: false,
+        targets: {
+          browsers: [
+            'Chrome >= 35',
```

Comment:
These are the minimum versions of browsers we support globally based on The Formula™

We can also add iOS, Android, and Edge here. Anything else?